python/ray/ml/config.py

```diff
@@ -1,8 +1,58 @@
-from typing import Dict, Any
+from dataclasses import dataclass
+from typing import Dict, Any, Optional, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ray.tune.trainable import PlacementGroupFactory
 
 # Right now, RunConfig is just an arbitrary dict that specifies tune.run
 # kwargs.
 # TODO(xwjiang): After Tuner is implemented, define the schema
 RunConfig = Dict[str, Any]
 
 ScalingConfig = Dict[str, Any]
+
+
+@dataclass
+class _ScalingConfigDataClass:
```

Comment:
```suggestion
class ScalingConfig:
```

Reply:
Never mind, had to undo this. ScalingConfig is already being used to define the type. When we make the dataclass the default representation, we can change the name accordingly.